ctx.go

```diff
@@ -17,13 +17,15 @@ import (
 	"net"
 	"net/http"
 	"path/filepath"
+	"reflect"
 	"strconv"
 	"strings"
 	"sync"
 	"text/template"
 	"time"
 
 	"github.com/gofiber/utils/v2"
+	"github.com/tiendc/go-deepcopy"
```

Comment:
Remove this, we dont need more external dependencies

Reply:
I need a way to deep copy

Reply:
We do not add extra dependencies unless they are really needed. This project only has ~50 stars and not many projects using it.

It also adds a ton of latency to the context object which would make Fiber even slower.

Reply:
Shoud I remove `DetachContext` emthod?

Reply:
Yes, thanks